File: member-service/src/main/java/com/hedvig/memberservice/commands/InitializeAppleUserCommand.java
package com.hedvig.memberservice.commands;

import lombok.Value;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

@Value
public class InitializeAppleUserCommand {

  @TargetAggregateIdentifier
  private long memberId;

  private String personalNumber;
  private String firstName;
  private String lastName;

  private String phoneNumber;
  private String email;

  private String street;
  private String city;
  private String zipCode;
}

File: member-service/src/main/java/com/hedvig/memberservice/commands/InitializeAppleUserCommand.kt
package com.hedvig.memberservice.commands

import org.axonframework.commandhandling.TargetAggregateIdentifier

class InitializeAppleUserCommand(
    @TargetAggregateIdentifier
    val memberId: Long,
    val personalNumber: String,
    val firstName: String,
    val lastName: String,
    val phoneNumber: String,
    val email: String,
    val street: String,
    val city: String,
    val zipCode: String
)
